app/src/main/java/org/fossasia/openevent/general/ticket/TicketsFragment.kt

```diff
@@ -10,13 +10,24 @@ import android.view.ViewGroup
 import android.widget.Toast
 import kotlinx.android.synthetic.main.fragment_tickets.view.*
 import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.event.EventUtils
+import org.fossasia.openevent.general.utils.nullToEmpty
 import org.fossasia.openevent.general.utils.Utils
 import org.koin.android.architecture.ext.viewModel
+import java.lang.StringBuilder
 
 class TicketsFragment : Fragment() {
     private val ticketsRecyclerAdapter: TicketsRecyclerAdapter = TicketsRecyclerAdapter()
     private val ticketsViewModel by viewModel<TicketsViewModel>()
     private var id: Long = -1
+    private lateinit var eventName: String
+    private var organizerName: String? = null
+    private lateinit var  startsAt: String
+    private lateinit var endsAt: String
```

Comment:
An event object would be better I think

Reply:
@dreadpool2 that's not possible. I am not loading these details from the server again. For that you need an entire event object but here we have only few fields of the event object. This case is a little different from how it is being done in the EventDetailsFragment.